### ISSUE TITLE
How to get more card data?

### ISSUE THREAD

**Issue by adamreisnz** [someone-noone/libnfc-js#2]
Could you tell me how I can get more information from a card/phone in the poll callback?
The `card` object passed to it only contains the following information, which is the same for both a plastic NFC card and my phone;

```
{ modulationType: 'NMT_ISO14443A', baudRate: 'NBR_106' }
```

I was hoping there would be some kind of unique identifier there or more information about the card/device.

**Comment by someone-noone** [comment]
well, I'm unsure if the library can provide more data. Could you please check whether nfc-poll command returns what you want. If yes, then I definitely could add the functionality. Just let me know what exactly you need.

**Comment by adamreisnz** [comment]
Yes, it does return more data, like card ID etc. In the end I switched to https://www.npmjs.com/package/nfc and it outputs much more information.

**Comment by someone-noone** [comment]
I see. The nfc package is for sure more mature project. However, it lacked the transceiver functionality and that's why I created this package. In any case, thank you for the interest and feedback. I'll try to do my best and update the library soon and include all data I can get from libnfc.

**Comment by adamreisnz** [comment]
Great, thank you. If you manage to get the data out of the card let me know and I'll have a look 👍
